test/quantization/core/test_docs.py

```diff
@@ -14,6 +14,16 @@
 from torch.testing._internal.common_utils import IS_ARM64
 
 
+@contextlib.contextmanager
+def use_qnnpack_qengine():
+    prev = torch._C._get_qengine()
+    torch._C._set_qengine(2)
```

Comment:
there is a public api for these I think, something like the following:
read: torch.backends.quantized.engine
write: torch.backends.quantized.engine = "qnnpack" 

you can also use https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_op.py#L521 to overwrite the quantized engine

Reply:
Cool, thanks!